onnxruntime/test/testdata/test_excluded_ops

```diff
@@ -0,0 +1,13 @@
+model:y
+"
```

Comment:
Please add something to the filename so it's clear why we have these files with no extension and what they contain. e.g. '.onnx.ignore_in_reduced_ops_build'

Reply:
Renamed like:
- onnx_model_of_excluded_ops
- onnx_model_of_included_ops
Feels a bit challenging to explain the reason of no extension by mere file name, add a readme?

Reply:
Further renamed to:

- reduced_ops_via_config.onnx_model_with_included_ops
- reduced_ops_via_config.onnx_model_with_excluded_ops

suggestion much appreciated!